packages/contracts/contracts/utils/MetaPtr.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+struct MetaPtr {
+
+  /// @notice Protocol ID corresponding to a specific protocol.
+  /// More info at https://github.com/gitcoinco/grants-round/tree/main/packages/contracts/docs/MetaPtrProtocol.md
+  uint256 protocol;
+  
+  /// @notice Pointer to fetch metadata for the specified protocol
+  string pointer;
+}
```

Comment:
@thelostone-mc I was thinking that we could use a URI instead of  MetaPtr, so that we don't need to define all the possible protocols and we can just set it to something like `ipfs://HASH` or `ceramic://did` etc /cc @kevinrolsen

Reply:
Which means at the client-side we'd have to split by prefix and check right ?
It feels like an extra step on the client side. 
We still aren't defining all the protocols, it's just a keeping it as a struct as opposed to a URI which needs to be

Reply:
@thelostone-mc cool it makes sense! they would be shared in between projects, so from grants-hub we could link to the same page.